websites/V/Viu/presence.ts

```diff
@@ -0,0 +1,84 @@
+const presence = new Presence({
+	clientId: "808404943311994910"
+}),
+strings = presence.getStrings({
+	play: "presence.playback.playing",
+	pause: "presence.playback.paused",
+	browsing: "presence.activity.browsing"
+}),
+video: HTMLVideoElement = document.querySelector("video#bitmovinplayer-video-vplayer"),
+timestamps = presence.getTimestamps(video.currentTime, video.duration);
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "viu-logo"
+	};
+	if (document.location.pathname.includes("video-")){
+		presenceData.startTimestamp = timestamps[0];
+		presenceData.endTimestamp = timestamps[1];
+		if (document.location.pathname.includes("_episode_")){
+			presenceData.details = document.querySelector("h1.ep_title").textContent.split("-")[0].replace(/\s*$/,'');
+			presenceData.state = document.querySelector("h1.ep_title").textContent.split("-")[1].replace(/^\s+|\s+$/g, "");
+		}else if (document.location.pathname.includes("trailer")){
+			if (document.querySelector("h1.ep_title").textContent.includes('\'')){
+				presenceData.details = document.querySelector("h1.ep_title").textContent.match(/'(.*?)'|"(.*?)"/)[1];
+			}else{
+				presenceData.details = document.querySelector("h1.ep_title").textContent.match(/'(.*?)'|"(.*?)"/)[2];
+			}
+			presenceData.state = "Trailer";
+		}else{
+			presenceData.details = document.querySelector("h1.ep_title").textContent;
+			presenceData.state = "Movie";
+		}
+		(presenceData.smallImageKey = video.paused ? "pause" : "play"),
+			(presenceData.smallImageText = video.paused ? (await strings).pause : (await strings).play),
+			(presenceData.startTimestamp = timestamps[0]),
+			(presenceData.endTimestamp = timestamps[1]);
```

Comment:
just do ; after each line and not with (),

Reply:
fixed